model/patch_lifecycle.go

```diff
@@ -41,14 +52,20 @@ func VariantTasksToTVPairs(in []patch.VariantTasks) []TVPair {
 // TVPairsToVariantTasks takes a list of TVPairs (task/variant pairs), groups the tasks
 // for the same variant together under a single list, and return all the variant groups
 // as a set of patch.VariantTasks.
-func TVPairsToVariantTasks(in []TVPair) []patch.VariantTasks {
+func TVPairsToVariantTasks(tasks []TVPair, displayTasks []TVPair) []patch.VariantTasks {
```

Comment:
should this be a method on model.TaskVariantPairs?

Reply:
yep, done